config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -686,6 +686,60 @@ periodics:
     testgrid-tab-name: cloud-provider-azure-master-vmss-capz
     testgrid-alert-email: kubernetes-provider-azure-oot@googlegroups.com
     description: "Runs Azure specific tests periodically with cloud-provider-azure (https://github.com/kubernetes-sigs/cloud-provider-azure) on vmss using a stable capz release."
+- cron: '0 18 * * *'  # Run at 18:00 UTC everyday
+  # cloud-provider-azure-vmssflex-capz runs Azure specific tests periodically on vmss flex.
+  name: cloud-provider-azure-vmssflex-capz
+  decorate: true
```

Comment:
Let's make it `optional: true`

Reply:
Looks like the "optional" cannot be distinguished by the check. Is is becuase"optional" is only avaialble for PR Job?

![image](https://user-images.githubusercontent.com/24629574/197656288-3c00755c-0c49-4f70-b337-d9e123147ef8.png)

Reply:
Makes sense.